src/components/Dropdown/__tests__/__snapshots__/dropdown-snapshot-tests.jest.js.snap

```diff
@@ -2215,12 +2213,11 @@ exports[`Dropdown renders correctly with leftIcon 1`] = `
         onTouchEnd={[Function]}
       >
         <svg
-          aria-hidden={false}
-          className="icon_component icon_component--clickable"
+          aria-hidden={true}
```

Comment:
clear indicator is a button so unless I missed something here we need to make sure we pass clickable=true and not the opposite (unless a container button is the actual button)

Reply:
Solved